server/routes/induction/common/affectAbilityToWorkController.ts

```diff
@@ -0,0 +1,44 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import type { InductionDto } from 'inductionDto'
+import type { AffectAbilityToWorkForm } from 'inductionForms'
+import InductionController from './inductionController'
+import AffectAbilityToWorkView from './affectAbilityToWorkView'
+
+/**
+ * Abstract controller class defining functionality common to both the Create and Update Induction journeys.
+ */
+export default abstract class AffectAbilityToWorkController extends InductionController {
+  constructor() {
+    super()
+  }
```

Comment:
This shouldn't be needed. 
```suggestion
```

Reply:
Yep, removed.

Btw, I was worried that might fail the lint check, so I've just re-committed it manually!